exampleSite/content/usage/configuration.md

```diff
@@ -134,6 +134,10 @@ params:
   # See also https://geekdocs.de/usage/menus/#bundle-menu
   geekdocMenuBundle: true
 
+  # (Optional, default false) Collapse all menu entries, can not be overwritten
+  # per page if enabled. Can be enabled per page via `geekdocCollapseSection`
+  GeekdocCollapseAllSections: true
```

Comment:
```suggestion
  geekdocCollapseAllSections: true
```

Reply:
why is this noted with lowercase here?

Reply:
Just to be consistent. Technically it's case-insensitive and doesn't matter, but we have started to write I lowercase in the front-matter and docs and uppercase in the themes code base, and it makes grepping a bit easier in some cases.

Reply:
thanks, i had issues grepping from config to code as the strings are different.